docs/how-to/apphook_configurations.rst

```diff
@@ -0,0 +1,110 @@
+.. _multiple_news_sections:
+
+#############################
+Multiple news/weblog sections
+#############################
+
+Your project can host multiple independent news/weblog sections, each with their own items. For
+example, a company website might have a news section for publishing press releases, and a weblog for
+publishing more informal articles.
+
+To do this, you need to create a django CMS page for each news/weblog section, and add an apphook
+for Aldryn News and Blog to each of them. You will also need to create a separate apphook
+configuration for each of them - apphook configurations cannot be shared between apphook instances.
+
+.. note:
+
+    Creating a new News & Blog section on your site implies setting up a new apphook instance.
+    The apphook instance however doesn't actually do anything until a page has been set up with it.
+
+
+**********************************
+Creating a new News & Blog section
+**********************************
+
+The quickest way to do this is:
+
+* Create the new page.
+* In its *Advanced settings*, choose *Newsblog* in the *Application* field. A new field,
+  *Application configurations*, will appear immediately below it.
+* Add a new application configuration, by selecting the **+** icon.
+
+
+Fields
+======
+
+*Instance namespace* - a unique name for this configuration. Note that this cannot be subsequently changed.
+
+*Application title* - A human-readable name for the configuration, that helps explain its purpose
+to the users of the system. For example, if this news section will publish press releases, call it
+*Press releases*. The name will be reflected in the django CMS toolbar when you're on that page.
+
+*Permalink type* - the format of canonical URLs for articles in this section.
+
+*Non-permalink-handling* - For convenience, the system can optionally resolve URLs that are not in
+the canonical format. For example, if the canonical URL is ``2016/11/27/man-bites-dog``, the URL
+``man-bites-dog`` can redirect to it. This behaviour is the default, but optional.
+
+*Prefix for template directories* - If you'd like this news section to use custom templates, create
+a set in a new directory. So for example, instead of using the default
+``aldryn_newsblog/article_list.html``, it will look for
+``aldryn_newsblog/custom-directory/article_list.html``.
+
+*Include in search index* - see :ref:`per_apphook_indexing`.
+
+Other fields are self-explanatory.
+
+Apphook configurations can also be created and edited in other ways:
+
+* from the Django Admin, in *Aldryn News and Blog* > *Configs*
```

Comment:
Shouldn't this PR also change the verbose name of the app-config from 'configs/' to 'News & Blog configurations' or 'application configurations'? If so, lines 145-146 in [project]/cms_appconfig.py.